Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestPage.tsx

```diff
@@ -128,24 +124,16 @@ const ContestPage = () => {
                         <SpinningLoader />
                     </div>
                 )
-                : isParticipationOfficial && contest?.isOnline && !isUserAdmin && !isUserParticipant
-                    ? (
-                        <ContestModal
-                          contest={modalContest}
-                          isShowing={isShowing}
-                          toggle={toggle}
-                        />
-                    )
+                : isParticipationOfficial && contest?.isOnline && !isUserAdmin && !startedModalUserParticipation
```

Comment:
I would reorder these so its easier to read:
1. contest?.isOnline - thats the most important thing that defines if we should show the modal so it should be first
2. isParticipationOfficial
3. !startedModalUserParticipation
4. !isUserAdmin

Reply:
Done.